datalad/interface/tests/test_add_archive_content.py

```diff
@@ -454,8 +459,11 @@ def test_add_delete_after_and_drop_subdir(self):
             commits_after_master = list(self.annex.get_branch_commits())
             commits_after = list(self.annex.get_branch_commits('git-annex'))
             # There should be a single commit for all additions +1 to
-            # initiate datalad-archives gh-1258
-            assert_equal(len(commits_after), len(commits_prior) + 2)
+            # initiate datalad-archives gh-1258.  If faking dates,
+            # there should be another +1 because annex.alwayscommit
+            # isn't set to false.
```

Comment:
I didn't write a good commit message in 33a7401cf (BF+TST: add_archive_content: Account for non-batched add_url_to_file, 2018-04-16) because, from reading it, I can't understand why the unbatched add_url_to_file call in add_archive_content causes issues with annex.alwayscommit=false. I don't recall if doing that would have required a large changes to add_archive_content, or if it was some deeper/general issue with unbatched/alwayscommit.  So, good question.  I'll have to look at it again.